integration-tests.yaml

```diff
@@ -26,6 +26,7 @@ steps:
     - 'TAG=${_TAG}-build-pipeline'
     - 'E2E_PROJECT_ID=${_E2E_PROJECT_ID}'
     - 'RUNTIME_BUILDER_ROOT=${_RUNTIME_BUILDER_ROOT}'
+    - 'TEST_VM_IMAGE=${_TEST_VM_IMAGE}'
```

Comment:
`build_images.sh` also uses this cloudbuild yaml so that `gcloud container builds submit` needs to add this substitution as well.